### DISCUSSION TITLE
Sound on Current release

### DISCUSSION THREAD

**Discussion by EzArclight** [dnschneid/crouton#5117] [Q&A]
I have managed to successfully install crouton, yet i find that there are  no working guides for getting pulse audio to function(or any guides for getting it fixed on aarch64) anyways, I am desperate to not use two devices, and i cant really use amazon for reasons. I do not know what details would be good, and i do not wish to clog, please lend me advice.

**Comment by EzArclight** [comment]
@dnschneid oh creator of this lovely software please. i have had this problem and i wish to solve it soon, I dont know if im allowed to do this but i do not care so im doing it anyways

**Reply by dnschneid** [reply]
those who tag my username in vain shall be smitten

...but in actuality I have no idea what modern ChromeOS is doing for audio.  The best suggestion I have is to follow [this guide](https://github.com/dnschneid/crouton/wiki/Access-audio-hardware-directly-(ALSA,-JACK)) (if you haven't tried it already) to enable direct hardware access.  I don't think you need to do the jack half of the steps since pulseaudio should pick up the alsa devices automatically.

**Reply by EzArclight** [reply]
okay, this did genuinly help, ive managed to get both the pulsa mixer working and bluetooth working, independently from eachother, 
now i just need to figure out how to have the audio play to my headphones
